stan/math/prim/prob/neg_binomial_2_cdf.hpp

```diff
@@ -48,18 +52,18 @@ return_type_t<T_location, T_precision> neg_binomial_2_cdf(
 
   VectorBuilder<!is_constant_all<T_precision>::value, T_partials_return,
                 T_precision>
-      digamma_phi_vec(size(phi));
-
-  VectorBuilder<!is_constant_all<T_precision>::value, T_partials_return,
+      digamma_phi_vec(size_phi);
+  VectorBuilder<!is_constant_all<T_precision>::value, T_partials_return, T_n,
```

Comment:
`T_n` has to be constant because it's an integer or array of integers, so it doesn't need to show up in `!is_constant`.

Reply:
Just to clarify: the  `T_n` is not inside `is_constant`, but among the typenames of the variables used in the `VectorBuilder`. So I think I should keep it.

Reply:
Thanks.  I initially misread where the closing `>` was.